### DISCUSSION TITLE
Truncating simulation length

### DISCUSSION THREAD

**Discussion by kinmic** [ss3sim/ss3sim#411] [General]
Hi there,

I'm currently working with ss3sim and trying to conduct some hindcasts. To do so, I've run a simulation 100 year simulation which will act as my observed values. I'm now trying to remove the terminal year from the OM (1:99) and predict the final year. I though I would be able to create a copy of the OM ('OM_trunc'), change the end year in the .dat file and then point the simulation to OM_trunc by adding the filepath in the 'om_dir' column in the simulation df. 

This doesn't seem to work and the simulation runs for 100 years. Do I need to remove the terminal line of data from all data sources? Or is there a more efficient method?

Thanks,

Mico

**Comment by kellijohnson-NOAA** [comment]
Feel free to email me your set up script (kelli.johnson@noaa.gov) but {ss3sim} can remove the terminal year of data for you if you just don't sample from that year and instead perform forecasts, e.g., `ce.forecast_num` column to set the number of forecast years in your estimation method.
